initialize_internal_tide.jl

```diff
@@ -112,7 +114,7 @@ v_immerse = ImmersedBoundaryCondition(bottom=immersed_drag_bc_v)
 
 u_bcs = FieldBoundaryConditions(bottom = drag_bc_u, top = FluxBoundaryCondition(nothing), immersed=u_immerse)
 v_bcs = FieldBoundaryConditions(bottom = drag_bc_v, top = FluxBoundaryCondition(nothing), immersed=v_immerse)
-
+w_bcs = FieldBoundaryConditions(immersed=ValueBoundaryCondition(0.0))  
```

Comment:
We need to remove this because it is incorrect. When there is variable topography, the bottom boundary condition is no-normal flow, which is only `w=0` for the bottom faces of the boundary-adjacent cells. For the boundary-adjacent west and east faces of a cell, it should be `u=0` instead, etc. The default BC on velocities is always no-normal flow, so we don't need to specify this.

Reply:
Ok, that makes sense.